crates/uv-cli/src/lib.rs

```diff
@@ -134,20 +134,39 @@ pub struct TopLevelArgs {
 #[command(next_help_heading = "Global options", next_display_order = 1000)]
 #[allow(clippy::struct_excessive_bools)]
 pub struct GlobalArgs {
-    /// Whether to prefer uv-managed or system Python installations.
+    #[arg(
+        global = true,
+        long,
+        help_heading = "Python options",
+        display_order = 700,
+        env = EnvVars::UV_PYTHON_PREFERENCE,
+        hide = true
+    )]
+    pub python_preference: Option<PythonPreference>,
+
+    /// Whether to require uv-managed Python installations.
```

Comment:
Similar to https://github.com/astral-sh/uv/pull/12246/files#r1999551644

```suggestion
    /// Only use uv-managed Python distributions.
```

(Sort of on the fence about "installations" vs "distributions" here but don't want users to be confused about whether we are "installing" here)

Reply:
I guess "versions" is actually the most consistent with other language?

Reply:
I wonder if we should also mention that this is equivalent to the `python-preference = "only-managed"` setting? I'm not sure. We might want to reference `python-preference` though unless we intend to later add a `managed-python = true | false` option to the settings to replace `python-preference`?

Reply:
I also think a `UV_MANAGED_PYTHON=1|0` variable might makes sense for these new options. It seems easier to use than `UV_PYTHON_PREFERENCE`.

Reply:
I'm hesitant to reference `python-preference` because the user might then try to learn two different ways to do the same thing (which might defeat part of the purpose of simplifying configuration here).

Reply:
The env var makes sense to me. I can add it.

Reply:
It would be nice to use the same language consistently. I'm fine with "versions".

Reply:
I considered an env var, but the problem is we have a third option as a default. 

We could use `UV_MANAGED_PYTHON` to mean 1 == required, 0 == not required (i.e., managed then system). But then we don't have "forbidden" represented. I didn't want another multi-value env var if we still have `UV_PYTHON_PREFERENCE`. But it also seemed a little awkward to add a `UV_NO_MANAGED_PYTHON` env var. 

What do you think?